Formula/pybind11.rb

```diff
@@ -47,5 +60,8 @@ def install
     python_flags = `#{Formula["python@3.9"].opt_bin}/python3-config --cflags --ldflags --embed`.split
     system ENV.cxx, "-O3", "-shared", "-std=c++11", *python_flags, "example.cpp", "-o", "example.so"
     system Formula["python@3.9"].opt_bin/"python3", "example.py"
+
+    system Formula["python@3.9"].opt_bin/"python3", "-m", "pybind11", "--includes"
+    system opt_bin/"pybind11-config", "--includes"
```

Comment:
Should we use an `assert_match` here to verify the output of this test?

Reply:
We could, it outputs `-I/usr/local/Cellar/python@3.9/3.9.1_4/Frameworks/Python.framework/Versions/3.9/include/python3.9 -I/usr/local/Cellar/pybind11/2.6.1/libexec/lib/python3.9/site-packages/pybind11/include` currently (and there's a small change we might swap `-I` for `-isystem` someday). Maybe the best thing to just verify that `libexec/site_packages` is in it?

Reply:
Yea, `assert_match "libexec/site_packages", ...` sounds good.

Reply:
Actually, should we also not try something like `import pybind11`, at least?

Reply:
Is there any way that `python -m pybind11` would work but `import pybind11` would not? Added tests for output.

Reply:
Good point; probably not. This is good enough then.